benchmarks/newton_solver_benchmark_set/spiegelman_et_al_2016/input.prm

```diff
@@ -0,0 +1,140 @@
+set Additional shared libraries            = ./libdrucker_prager_compositions.so
+
+set Use years in output instead of seconds = false  # default: true
+set Nonlinear solver scheme = Newton Stokes
+set Max nonlinear iterations = 150
+set Nonlinear solver tolerance = 1e-14
+set Pressure normalization = no
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Number of cheap Stokes solver steps = 0
+    set Linear solver tolerance = 1e-5
+  end 
+  subsection Newton solver parameters
+        set Max pre-Newton nonlinear iterations = 5
+    set SPD safety factor = 0.9
+    set Nonlinear Newton solver switch tolerance = 1e-20 #  
+        set Max Newton line search iterations = 0
+    set Maximum linear Stokes solver tolerance = 9e-1
+        set Use Newton residual scaling method = false
+    set Use Newton failsafe = false
+    set Stabilization preconditioner = SPD
+    set Stabilization velocity block = SPD
+  end
+end
+
+subsection Boundary temperature model
+  set Model name = box
+end
+
+
+ subsection Initial temperature model
+   set Model name = function
+ end
+
+ subsection Initial composition model
```

Comment:
Can you add a brief description here of what the equations below produce (i.e., square with rounded edges)?